index.bs

```diff
@@ -171,32 +172,19 @@ Resource names starting with U+002D HYPHEN-MINUS (-) are reserved; requesting th
 ## Lock Managers ## {#lock-managers}
 <!-- ====================================================================== -->
 
-A [=/user agent=] has a <dfn>lock manager</dfn> for each [=/origin=], which encapsulates the state of all [=lock-concept|locks=] and [=lock requests=] for that origin.
+A [=/user agent=] has a <dfn>lock manager</dfn> for each [=/storage bucket=], which encapsulates the state of all [=lock-concept|locks=] and [=lock requests=] associated with a [=/storage bucket=].
```

Comment:
Each bucket has bottles for each storage endpoint, so should this be storage bottle instead as Anne pointed out below?

Reply:
I think the reason I left this as "bucket" is that there is other (non-normative?) text that tries to describe that locks are intended for coordination across agents that share a storage bucket (including in a future world with multiple buckets per origin/storage key), so (at the time) I wanted to keep the bucket term here as context. But it sounds like that may be confusing for someone reading this then looking at the algorithm, even though it's a 1:1 relationship.

How about...

> "A lock manager encapsulates the state of locks and lock requests. A lock manager is associated with a storage bottle for the Web Locks API local storage endpoint in each storage bucket."

Further wordsmithing is appreciated!

Reply:
That phrase sounds better to me! One nit is that, the 1:1 relationship of the bottle/bucket and the lock manager is less clear than the current phrase.

My try:

> "A lock manager encapsulates the state of locks and lock requests. Each storage bucket includes one lock manager through an associated storage bottle for the Web Locks API.

Does this sound better to you? 👀🤔 This way I feel like the relationship between the bucket and bottle is less clear, but that specific relationship is technically not in the scope of this spec...